conda-store-server/tests/test_db_api.py

```diff
@@ -71,6 +71,108 @@ def test_namespace_role_mapping(db):
     NamespaceRoleMapping(namespace=namespace, namespace_id=namespace.id, entity="*/*")
 
 
+def test_namespace_role_mapping_v2(db):
+    namespace_name = "pytest-namespace"
+    other_namespace_name1 = "pytest-other-namespace1"
+    other_namespace_name2 = "pytest-other-namespace2"
+    other_namespace_name3 = "pytest-other-namespace3"
+
+    # Starts with no namespaces
+    assert len(api.list_namespaces(db).all()) == 0
```

Comment:
It might be simpler to just clear out the namespaces like you do in the auth tests than to assert here.